### PR TITLE
kcadm.sh: add page

### PR DESCRIPTION
**Pull request by thamaraiselvam**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/kcadm.sh.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# kcadm.sh
+
+> With Keycloak, you can perform administration tasks from the command-line interface (CLI) by using the Admin CLI command-line tool.
```
Comment (kbdharun) [inline pages/common/kcadm.sh.md:3]:
```suggestion
> Perform administration tasks from the command-line interface (CLI).
```

File: pages/common/kcadm.sh.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://www.keycloak.org/docs/latest/server_admin/#admin-cli>.
+
+- Start an authenticated session:
+
+`kcadm.sh config credentials --server {{host}} --realm {{realm_name}} --user {user_name} --password {{password}}`
```
Comment (kbdharun) [inline pages/common/kcadm.sh.md:8]:
```suggestion
`kcadm.sh config credentials --server {{host}} --realm {{realm_name}} --user {{username}} --password {{password}}`
```

File: pages/common/kcadm.sh.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Create a user:
+
+`kcadm.sh create users -s username={{user_name}} -r {{realm_name}}`
```
Comment (kbdharun) [inline pages/common/kcadm.sh.md:12]:
```suggestion
`kcadm.sh create users -s username={{username}} -r {{realm_name}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Comment by thamaraiselvam** [comment]
Thanks for your suggestions @kbdharun. I have resolved them.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**